hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/MulticastService.java

```diff
@@ -283,7 +283,14 @@ public void send(JoinMessage joinMessage) {
                 multicastSocket.send(datagramPacketSend);
                 out.clear();
             } catch (IOException e) {
-                logger.warning("You probably have too long Hazelcast configuration!", e);
+                // usually catching EPERM errno
+                // see https://github.com/hazelcast/hazelcast/issues/7198
+                // For details about the causes look at the following discussion:
+                // https://groups.google.com/forum/#!msg/comp.protocols.tcp-ip/Qou9Sfgr77E/mVQAPaeI-VUJ
+                logger.warning("Sending multicast datagram failed. Exception message saying the operation is not permitted "
```

Comment:
It can also be caused if the payload is larger than 64KB.

Reply:
Our join datagram packets have size around 300B IIRC.

Reply:
In the past the hz config was included in the payload. I guess that isn't the case any longer?